src/libs/ValidationUtils.js

```diff
@@ -321,6 +321,34 @@ function doesFailCharacterLimit(maxLength, valuesToBeValidated) {
     return _.map(valuesToBeValidated, value => value.length > maxLength);
 }
 
+/**
+ * Checks if is one of the certain names which are reserved for default rooms
+ * and should not be used for policy rooms.
+ *
+ * @param {String} roomName
+ * @returns {Boolean}
+ */
+function isReservedRoomName(roomName) {
+    return _.contains(CONST.REPORT.RESERVED_ROOM_NAMES, roomName);
+}
+
+/**
+ * Checks if the room name already exists. We don't care if it matches the original name provided in this
+ * component because then we are not changing the room's name
+ *
+ * @param {String} roomName
+ * @param {Object[]} reports
+ * @param {String} policyID
+ * @returns {Boolean}
+ */
+function isExistingRoomName(roomName, reports, policyID) {
+    return _.some(
+        _.values(reports),
```

Comment:
`_.some()` should work on object collections so no need to use `_.values()` here 

https://underscorejs.org/#some